packages/module-services/src/bookmarks/get/generate-endpoint.ts

```diff
@@ -0,0 +1,22 @@
+import { UnsupportedApiVersion } from '@equinor/fusion-framework-module-services/errors';
+import { ApiVersions, GetBookmarkArgs } from './types';
+
+/**
+ * Method for generating endpoint for getting bookmark by id
+ */
+export const generateEndpoint = <TVersion extends ApiVersions>(
+    version: TVersion,
+    args: GetBookmarkArgs<TVersion>
+) => {
+    switch (version) {
+        case 'v2':
+            throw new UnsupportedApiVersion(version);
```

Comment:
just remove since there are no options to select this, it was included in `@equinor/fusion-framework-module-services/context` since it actually has a v2, and was left as a placeholder for howto map/handle multiple versions